### ISSUE TITLE
Integrated Remote.SSH replacement doesn't seem to be picking up the correct ssh binary 

### ISSUE THREAD

**Issue by ELLIOTTCABLE** [getcursor/cursor#1054]
Attempting to try out Cursor, and imported my VScode config wholesale. Unfortunately, immediately blocked by being unable to connect to the devboxes we use every day:

<details><summary>Screenshot</summary>

<img width="1112" alt="Screenshot 2023-11-07 at 12 24 12" src="https://github.com/getcursor/cursor/assets/200/875a6ddd-5fb4-49d3-bdbd-7d03015bb2bf">

</details>

Glancing at the config, it appears (hard to tell, no `remote.SSH.logLevel` to configure …) that perhaps it's using the wrong `ssh` binary? We use [`ssh-ident`](https://github.com/ssh-ident/ssh-ident1) to isolate private-keys to specific hosts, so on my local machine, the correct `ssh` binary is in the home-dir: `/Users/ec/.bin/ssh`.

VScode picks this up fine, but for some surprising reason, Cursor's fork dosen't seem to be — it's in the `$PATH` (although, what's "in the `$PATH` is, of course, a surprisingly complex topic, heh …).

Relatedly, I can't work around this, due to the lack of a correspondent to `remote.SSH.path`.

**Comment by ELLIOTTCABLE** [comment]
Further relevant details:

1. The effective ssh_config for the `nspawn` will be computed to be something similar to …

   ```
   Host febox-us
     Hostname <SNIP>
     User elliott.cable
     IdentityFile ~/.ssh/ahrefs/identity

   Host nspawn
     User me
     Hostname <SNIP>
     ProxyJump elliott.cable@febox-us
 
     User elliott.cable
     IdentityFile ~/.ssh/ahrefs/identity
     ForwardAgent yes
     StreamLocalBindUnlink yes
   ```

2. We use a lot of Wireguard tunneling, etc, which may be complicating the scenario.

Again, though, hopefully, none of that should matter, as this works in upstream VScode-remote-ssh — and has for a good long while, so I don't think it's any recent fixes that you haven't pulled down yet? 💀

**Comment by leerob** [comment]
Thanks for opening this issue! We've moved our community support to the [Cursor Forum](https://forum.cursor.com) where you can get help from both the team and other users.

If this is a specific bug that you can still reproduce, please open a new issue with clear reproduction steps on the forum.

Thanks you!
